### PR TITLE
Documentation: Added options and negative options to manpages

### PR DESCRIPTION
**Pull request by nangahamandine**
Updated the ocamlopt.1, ocaml.1 and ocamlc.1 files with missing negative options in man directory

### CODE REVIEW

File: man/ocaml.1
Comment (Octachron) [inline man/ocaml.1]: Main options should be sorted by alphabetical order, thus `strict-format` should be just before `strict-sequence`.

File: man/ocaml.1
Comment (Octachron) [inline man/ocaml.1]: I would rather have this change in a separate PR because the text in the development branch and in the 5.0 branch needs to be different.

File: man/ocaml.1
Comment (Octachron) [inline man/ocaml.1]: The REPL does not really create interface that are reused by other users. It seems better to keep this note out of this specific description.

File: man/ocamlopt.1
Comment (Octachron) [inline man/ocamlopt.1]: This spurious change is here because you created this branch from the first version of your previous PR, before we moved the location of the flambda section. To remove this change, you can either try to rebase your branch on trunk or remove this change by hand in a commit (that will disappear when this PR is squashed into a single commit).

File: man/ocaml.1
Comment (nangahamandine) [reply man/ocaml.1]: Would you like me to remove the changes made in the ocaml.1 file and change back it to the default?

Comment (Octachron) [reply man/ocaml.1]: I was speaking of the `-strict-format` and `-no-strict-format` change rather than the changes in `ocaml.1`.

Comment (nangahamandine) [reply man/ocaml.1]: Oh alright. So I'll need to take out the -strict-format and -no-strict-format changes made in all three files for now and later create a separate PR to add the changes, right?

Comment (Octachron) [reply man/ocaml.1]: Indeed. Moreover you would need to have one version of the PR for the `5.0` branch and another for the `trunk` branch because `-strict-format` is now the default on the `trunk` branch.

**Comment by nangahamandine** [comment]
Are the recent changes for this PR okay

File: man/ocamlopt.1
Comment (Octachron) [inline man/ocamlopt.1]: This note should neither be removed from `ocamlc.1` and `ocamlopt.1` nor added to `ocaml.1`.

File: man/ocamlopt.1
Comment (Octachron) [inline man/ocamlopt.1]: This section is now repeated twice at the same location, you should remove it.

**Review by Octachron — APPROVED** [review]
This looks good, thanks!
You still need to add a Change entry. I would suggest that you merge the entry with the one from the previous PR:

```
- #11640, #11647: Add missing options to the man pages:
  flambda commonly-used options, and negative options (`-no-rectypes`, ... ).
  (Amandine Nangah, review by David Allsopp, Florian Angeletti,
  Sébastien Hinderer, and Vincent Laviron)
```

File: man/ocaml.1
Comment (Octachron) [inline man/ocaml.1]: It would be better to use `\-no\-rectypes`.

File: man/ocaml.1
Comment (Octachron) [inline man/ocaml.1]: Same as above `\-no\-strict\-sequence`.

File: man/ocamlc.1
Comment (Octachron) [inline man/ocamlc.1]: Same as above `\-no\-principal`.

File: man/ocamlopt.1
Comment (Octachron) [inline man/ocamlopt.1]: You are missing a `.TP` line here.

**Review by Octachron — CHANGES_REQUESTED** [review]
Before merging, there are few formatting typos left that should be fixed.

File: man/ocamlopt.1
Comment (Octachron) [inline man/ocamlopt.1]: One last missing `\` here: `\-no\-rectypes`.

**Review by Octachron — APPROVED** [review]

**Comment by nangahamandine** [comment]
Can I go ahead to create another PR for the other changes? I was waiting for approval before making other changes

**Comment by Octachron** [comment]
You can certainly start working on another PR. You might want to try your hand at a non-documentation issue (maybe #11127 ?).

**Comment by nangahamandine** [comment]
Okay, thanks
I was hoping to complete issue #6955 before moving to another
I've been working on subsets. So if it's okay, may I complete the entire issue first?

**Comment by Octachron** [comment]
It would be a better use of your application period to try your hand at more challenging tasks. 
(You will be of course very welcome to come back to the issue after the Outreachy application period.)

**Comment by nangahamandine** [comment]
Okay then

**Comment by Octachron** [comment]
Merged, thanks again!

**Pull request merged**
